platform/ios/ios.xcodeproj/xcshareddata/xcschemes/Integration Test Harness.xcscheme

```diff
@@ -36,6 +36,13 @@
             ReferencedContainer = "container:ios.xcodeproj">
          </BuildableReference>
       </MacroExpansion>
+      <EnvironmentVariables>
+         <EnvironmentVariable
+            key = "MAPBOX_ACCESS_TOKEN"
+            value = "pk.eyJ1IjoianVsaWFucmV4IiwiYSI6ImNrMGp2N3k3ZzAwbWkzbnMzcG1zZ291cmkifQ.6Fv-dxHmzcqHHm_Tlf3x6A"
```

Comment:
Do we want to hard code a real access token here? Can we use the same script that iosapp uses to pull in the access token from `~/.mapbox`?

Reply:
Oops! Yeah, that shouldn't be there. Can you remove please?

Reply:
For future reference - this access token has been revoked.